add analysis layout

ToboMasaya opened the pull request:
とりあえずの分析画面
![analysis_layout](https://user-images.githubusercontent.com/39518731/60401680-25b29f00-9bc0-11e9-8ff8-24f2ace33693.png)

ToboMasaya committed:
分析画面、とりあえず

pikohan-suzuki commented on app/src/main/res/layout/analysis_layout.xml:
```diff
@@ -0,0 +1,48 @@
+                                             android:layout_height="match_parent">
+
+    <TextView
+            android:text="最低危険度　　　%"
```
"最低危険度 %1$d %%"といった風に後から数値を代入しやすい形にしたほうがいいと思います。

pikohan-suzuki committed:
add constraint

pikohan-suzuki commented:
"最低危険度 %1$d %%"といった表記に修正しました。
適当なid付けました。
下半分のタスクのリストはRecyclerViewに置き換えました。

pikohan-suzuki commented:
まだ、分析画面のレイアウトについて十分話し合ってないので、火曜日にみんなのイメージをすり合わせませう

Bluewind12 commented on app/src/main/res/layout/analysis_layout.xml:
```diff
@@ -0,0 +1,56 @@
+            android:id="@+id/max_yabasa" android:textSize="30sp"
+            android:textColor="#FF0000" android:textAlignment="center" android:layout_marginStart="8dp"
+            app:layout_constraintStart_toStartOf="parent" android:layout_marginEnd="8dp"
+            app:layout_constraintEnd_toEndOf="parent" android:layout_marginTop="92dp"
```
android:layout_marginTop="92dp"
8の倍数の方が後で楽

ToboMasaya replied:
了解です

Bluewind12 commented on app/src/main/res/layout/analysis_layout.xml:
```diff
@@ -0,0 +1,56 @@
+            android:textColor="#FF0000" android:textAlignment="center" android:layout_marginStart="8dp"
+            app:layout_constraintStart_toStartOf="parent" android:layout_marginEnd="8dp"
+            app:layout_constraintEnd_toEndOf="parent" android:layout_marginTop="92dp"
+            app:layout_constraintTop_toTopOf="parent" app:layout_constraintHorizontal_bias="0.468"
```
app:layout_constraintHorizontal_bias="0.468"
とかこの辺は、数値キリよくしといた方が見栄え良くなる

ToboMasaya replied:
ありがとうございます。

Bluewind12 reviewed (commented):
適当に見栄えの問題だからそんなでもないけども

pikohan-suzuki commented on app/src/main/res/layout/analysis_layout.xml:
```diff
@@ -0,0 +1,56 @@
+
+    <TextView
+            android:text="@string/min_yabasa"
+            android:layout_width="369dp"
```
テストです。

ToboMasaya committed:
切りのよい数字に変更しました

pikohan-suzuki commented:
いいとおもいます。

Bluewind12 reviewed (commented):
ID,Colorの変更した方がいいのでは？
ID「TextView4」
カラー：リソースから引っ張るように。

ToboMasaya committed:
id名などをまとめました。

最高難易度 max_risk
平均難易度 avg_risk
最小難易度 min_risk
未完了タスク non_complete_task
guideline ⇒ analysis_guideline

Bluewind1997 committed:
Merge branch 'develop' into layout_analysis
